platform/mds/mds/src/main/resources/db/migration/default/V6__MOTECH-2578.sql

```diff
@@ -0,0 +1,22 @@
+-- Added auto increment sequence
+
```

Comment:
@pgesek 
I looked for solution with adding serial type. I found some solution which works, when we add a new field in the table, but when we want to change type in an existing field, it doesn't work.